integrated_channels/api/v1/sap_success_factors/serializers.py

```diff
@@ -26,3 +26,11 @@ class Meta:
 
     encrypted_key = serializers.CharField(required=False, allow_blank=False, read_only=False)
     encrypted_secret = serializers.CharField(required=False, allow_blank=False, read_only=False)
+
+
+class SAPUserInfoRequestSerializer(serializers.Serializer):
+    """
+    Serializer for SAP user info request parameters.
+    """
+    org_id = serializers.CharField(required=True)
+    loggedinuserid = serializers.CharField(required=True)
```

Comment:
nit: Can we align the variable naming conventions here? The first uses underscores, while the second does not.